addon/ng2/commands/test.ts

```diff
@@ -14,7 +14,7 @@ module.exports = TestCommand.extend({
   ],
 
   run: function (commandOptions) {
-    this.project.ngConfig = this.project.ngConfig || config.CliConfig.fromProject();
+    this.project.ngConfig = this.project.ngConfig || CliConfig.fromProject();
```

Comment:
Weird, I thought I saw this being attributed loads of times.. but it's only in `index.js` now that I double check. My bad.